fixtures/coverall-upstream-compatibility/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "uniffi-fixture-coverall-upstream-compatibility"
+version = "0.22.0"
+authors = ["Firefox Sync Team <sync-team@mozilla.com>"]
+edition = "2021"
+license = "MPL-2.0"
+publish = false
+
+[lib]
+crate-type = ["lib", "cdylib"]
+name = "uniffi_coverall_upstream_compatibility"
+
+[dependencies]
+uniffi = { path = "../../uniffi" }
```

Comment:
I think this should also point to the upstream repository. `dependencies` and `build-dependencies` both refer to scaffolding code. `build-dependencies` is used to generate scaffolding code, but the scaffolding code needs to be compiled and it depends on uniffi crates, so `dependencies` also need to be updated. I guess its not a major issue, but I think these should be consistent for the sake of correctness.

Reply:
Nope. Only `build-dependencies` are used in build step. On the other hand when you build tests and use `dev-dependencies` you also use `dependencies`

Reply:
This is a little different from normal usage though, because in our projects we usually use uniffi::include_scaffolding. But in those tests the scaffolding is generated in build step for the fixture and then included in tests from the file by include macro

Reply:
Yes, I completely forgot that the scaffolding itself contains references to the uniffi. It is now fixed and should behave as expected. Thanks!